modules/caddyhttp/rewrite/rewrite.go

```diff
@@ -89,6 +89,10 @@ type Rewrite struct {
 	// Performs regular expression replacements on the URI path.
 	PathRegexp []*regexReplacer `json:"path_regexp,omitempty"`
 
+	// Mutates the query string of the URI.
+
+	Query *queryOps `json:"query_operations,omitempty"`
```

Comment:
Shouldn't be a space in between, otherwise the comment won't be seen as a Godoc comment when generating the docs.

Also I think we should just shorten this to `query`, the `_operations` doesn't really add much to the config output. The fact that we're doing an "operation" is clear enough by the verbs like "add/set/rename/delete".

```suggestion
	// Mutates the query of the URI by applying a set of operations.
	Query *queryOps `json:"query,omitempty"`
```